pkgs/by-name/ag/age/package.nix

```
@@ -12,44 +13,42 @@
   age-plugin-1p,
   makeWrapper,
   runCommand,
+  versionCheckHook,
+  nix-update-script,
 }:
 
 buildGoModule (final: {
   pname = "age";
-  version = "1.2.1";
+  version = "1.3.0";
 
   src = fetchFromGitHub {
     owner = "FiloSottile";
     repo = "age";
     tag = "v${final.version}";
-    hash = "sha256-9ZJdrmqBj43zSvStt0r25wjSfnvitdx3GYtM3urHcaA=";
+    hash = "sha256-ToKLdLVi4tqWa3iXLLTYbjyYJv/QUun5ysEeVuiEni4=";
   };
 
-  vendorHash = "sha256-ilRLEV7qOBZbqzg2XQi4kt0JAb/1ftT4JmahYT0zSRU=";
+  vendorHash = "sha256-iVDkYXXR2pXlUVywPgVRNMORxOOEhAmzpSM0xqSQMSQ=";
+
+  patches = [
+    # patch out debug.ReadBuidlInfo since version information is not available with buildGoModule
```

```suggestion
    # patch out debug.ReadBuildInfo since version information is not available with buildGoModule
```

I also just... don't understand the comment "version information is not available with buildGoModule"

---

I removed the linker variable upstream because debug.ReadBuildInfo now stamps the version from the VCS, but didn't realize it'd break packagers. Kinda obvious in retrospect, sorry. Will make v1.3.1 with Version again.

---

Fixed in v1.3.1. Sorry and thanks for packaging age.

---

Thanks for the quick fix! Updated to 1.3.1 and removed the patch